plugins/catalog-backend/src/ingestion/processors/github/github.ts

```diff
@@ -63,6 +64,18 @@ export type Connection<T> = {
   nodes: T[];
 };
 
+/**
+ * Gets all GitHub organizations the app is installed in.
+ *
+ * @param client An octokit REST client for a GitHub app
+ */
+export async function getOrganizations(client: Octokit) {
+  const installations = await client.paginate(client.apps.listInstallations);
```

Comment:
I'm no expert here, but I believe the default Octokit library only calls GitHub.com.  Is there anything we need to pass in here for this to support GHE?

Reply:
So I think this method should expect the passed in Octokit client to already be configured correctly to point to a GHE instance which I do here: https://github.com/backstage/backstage/pull/5173/files#diff-18d2ea790aeb3068a8e8abc3907e72e37617eef845e0a02bfdf24086d3abca50R60